CalibTracker/SiStripHitEfficiency/plugins/SiStripHitEfficiencyHarvester.cc

```diff
@@ -213,38 +270,98 @@ void SiStripHitEfficiencyHarvester::dqmEndJob(DQMStore::IBooker& booker, DQMStor
     const auto num = h_module_found->getValue(det);
     const auto denom = h_module_total->getValue(det);
     if (denom) {
-      const auto eff = num / denom;
-      hEffInLayer[layer]->Fill(eff);
-      if (!autoIneffModTagging_) {
-        if ((denom >= nModsMin_) && (eff < threshold_)) {
-          // We have a bad module, put it in the list!
-          badModules[det] = eff;
-          tkMapBad.fillc(det, 255, 0, 0);
-          LOGPRINT << "Layer " << layer << " (" << ::layerName(layer, showRings_, nTEClayers_) << ")  module "
-                   << det.rawId() << " efficiency: " << eff << " , " << num << "/" << denom;
-        } else {
-          //Fill the bad list with empty results for every module
-          tkMapBad.fillc(det, 255, 255, 255);
+      // use only the "good" modules
+      if (stripQuality_->getBadApvs(det) == 0) {
+        const auto eff = num / denom;
```

Comment:
if the `auto` for the `num` and `denom` is an `uint`, will the `auto` for the `eff` somehow take care of the integer division?

Reply:
The code was already like that before, and as far as I can tell does the right thing.